polar-core/src/polar.lalrpop

```diff
@@ -463,13 +466,13 @@ pub Value: Value = {
     <BuiltinOperation>,
     <RewrittenOperation>,
     <Predicate>,
-    "[" <mut terms:(<Term> ",")*> <term:Spanned<RestVar<"Term">>?> "]" => {
-        match term {
-            Some(term) => {
-                terms.push(term);
-                Value::List(terms)
+    "[" <mut head:(<Term> ",")*> <last:Spanned<RestVar<"Term">>?> "]" => {
```

Comment:
nit: I'd prefer this as `head` + `tail`

Reply:
Yes, but punt and it gets changed in #390 anyway